packages/blocks/src/embed-synced-doc-block/embed-synced-doc-block.ts

```diff
@@ -268,14 +268,23 @@ export class EmbedSyncedDocBlockComponent extends EmbedBlockElement<
 
       const edgeless = this.edgeless;
       assertExists(edgeless);
-      const blockId = edgeless.service.addBlock(
+      const newId = edgeless.service.addBlock(
         'affine:embed-linked-doc',
         { pageId, xywh: bound.serialize(), style, caption },
         edgeless.surface.model
       );
+
+      this.std.command
```

Comment:
If you just want to call a single command, you can use `command.exec(commandName, commandOptions)`

Reply:
Similar to this: https://github.com/toeverything/blocksuite/pull/7200/files